org.eclipse.xtext/src/org/eclipse/xtext/validation/Issue.java

```diff
@@ -43,17 +43,31 @@
 	 * Returns the one-based line number of the issue.
 	 */
 	Integer getLineNumber();
+
+	/**
+	 * Returns the one-based line number of the end of the issue.
```

Comment:
Please also state that a value '0' indicates 'no line information'

Reply:
AFAIR, we said the contract was: '<1' indicates 'no information'

Reply:
:+1:

Reply:
`Values smaller than 1 are invalid and indicate the absence of line information on this issue.`